src/app/cart-item/cart-item.component.html

```diff
@@ -0,0 +1,14 @@
+<ng-template [ngIf]="cartItem">
+  <li class="item-details">
+    <img
+      class="cart--item-icon"
+      [src]="generateImagePath(cartItem[0])"
```

Comment:
It would be best if we could avoid magic numbers in the template as this would cause an error if we didn't have an element in the array

Reply:
I can see that you're sending two bits of data through the input, and that's why you have it as a tuple. 

I think it would have been better to change the Item data structure so that both bits of information were stored in the same object.